src/argilla_server/apis/v1/handlers/users.py

```diff
@@ -21,12 +21,20 @@
 from argilla_server.database import get_async_db
 from argilla_server.models import User
 from argilla_server.policies import UserPolicyV1, authorize
+from argilla_server.schemas.v1.users import User as UserSchema
 from argilla_server.schemas.v1.workspaces import Workspaces
 from argilla_server.security import auth
 
 router = APIRouter(tags=["users"])
 
 
+@router.get("/me", response_model=UserSchema)
+async def get_current_user(current_user: User = Security(auth.get_current_user)):
+    # TODO: Should we add telemetry.track_login?
```

Comment:
I would say to replicate the same logic as the old endpoint. So, it makes sense to include also here.